Vagrantfile

```diff
@@ -6,7 +6,7 @@ Vagrant.configure('2') do |config|
   config.disksize.size = '50GB'
   config.vm.box_check_update = false
   config.vm.host_name = 'sumologic-tailing-sidecar'
-  config.vm.network :private_network, ip: "192.168.78.67"
+  config.vm.network :private_network, ip: "192.168.56.87"
```

Comment:
Can you add the same comment we've been adding everywhere else about this IP address: https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/4bf6046636bc65a5dc487db8953318f78cfe6e62/Vagrantfile#L15?

Reply:
OK done